pkg/gadgets/trace/tcpretrans/tracer/tracer.go

```diff
@@ -147,6 +154,14 @@ func (t *Tracer) run() {
 
 		ipversion := gadgets.IPVerFromAF(bpfEvent.Af)
 
+		typ := "unknown"
+		switch bpfEvent.Type {
+		case tcpretransTypeRETRANS:
+			typ = "retrans"
+		case tcpretransTypeLOSS:
+			typ = "loss"
+		}
+
```

Comment:
image-based gadgets use uppercase for enums. This gadget also does use uppercase for the tcp state.